api/v1alpha1/tenant_types.go

```diff
@@ -38,6 +38,13 @@ type IngressClassesSpec struct {
 	AllowedRegex string           `json:"allowedRegex,omitempty"`
 }
 
+type IngressHostnamesSpec struct {
+	// +nullable
```

Comment:
let's remove the `nullable` marker here: we don't need it if we mark the `IngressHostnamesSpec` as `omitempty`: refer to this.

https://github.com/clastix/capsule/blob/98e441f1e9a9e24630624fcb8c68ff5039b59e28/api/v1alpha1/tenant_types.go#L61-L63

https://github.com/clastix/capsule/blob/98e441f1e9a9e24630624fcb8c68ff5039b59e28/api/v1alpha1/tenant_types.go#L31-L44

Reply:
done.